scripts/checkpatch_inc.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+CHECKPATCH="${CHECKPATCH:-checkpatch.pl}"
+# checkpatch.pl will ignore the following paths
+CHECKPATCH_IGNORE=$(echo \
```

Comment:
This part bring in things not relevant to optee_test.

Reply:
Yes, but doesn't seem to hurt either. I simply copied the files from the optee_os repo. Unless there is a pressing reason to branch off the code at this point, I'd suggest keeping it that way. For the time being this helps with maintenance issues. A long term goal could be to move the checkpatch (and probably other developer support scripts) to its own repo, so it can be re-used as a submodule at multiple places in the codebase and for TA development.